Submitted By: Nathan Coulson
Date: 2004-03-27
Initial Package Version: 1.4.2
Origin: blfs-support
Description: Allows compilation if /bin/sh is linked to /bin/ash by calling bash instead

$LastChangedBy: $
$Date: $

diff -Naur j2sdk-1.4.2.OLD/hotspot/build/linux/Makefile j2sdk-1.4.2/hotspot/build/linux/Makefile
--- j2sdk-1.4.2.OLD/hotspot/build/linux/Makefile	2003-09-10 19:40:03.000000000 -0600
+++ j2sdk-1.4.2/hotspot/build/linux/Makefile	2004-02-24 00:10:35.000000000 -0700
@@ -71,7 +71,7 @@
 	chmod +x ./test; ./test
 
 ${OS}_${ARCH}_compiler2/debug ${OS}_${ARCH}_compiler2/optimized ${OS}_${ARCH}_compiler2/fastdebug ${OS}_${ARCH}_compiler2/jvmg ${OS}_${ARCH}_compiler2/profiled ${OS}_${ARCH}_compiler2/product : $(buildATree)
-	sh $(buildATree) compiler2 ${GAMMADIR} ${OS} ${ARCH}
+	bash $(buildATree) compiler2 ${GAMMADIR} ${OS} ${ARCH}
 
 debug: ${OS}_${ARCH}_compiler2/debug
 	(cd ${OS}_${ARCH}_compiler2/debug; $(MAKE))
@@ -98,7 +98,7 @@
 #	@(cd ${OS}_${ARCH}_compiler2/profiled; gamma=gamma; $(RUN_TEST))
 
 ${OS}_${ARCH}_compiler1/debug ${OS}_${ARCH}_compiler1/optimized ${OS}_${ARCH}_compiler1/fastdebug ${OS}_${ARCH}_compiler1/jvmg ${OS}_${ARCH}_compiler1/profiled ${OS}_${ARCH}_compiler1/product : $(buildATree)
-	sh $(buildATree) compiler1 ${GAMMADIR} ${OS} ${ARCH}
+	bash $(buildATree) compiler1 ${GAMMADIR} ${OS} ${ARCH}
 
 debug1: ${OS}_${ARCH}_compiler1/debug
 	(cd ${OS}_${ARCH}_compiler1/debug; $(MAKE))
@@ -125,7 +125,7 @@
 #	@(cd ${OS}_${ARCH}_compiler1/profiled; gamma=gamma; $(RUN_TEST))
 
 ${OS}_${ARCH}_core/debug ${OS}_${ARCH}_core/optimized ${OS}_${ARCH}_core/fastdebug ${OS}_${ARCH}_core/jvmg ${OS}_${ARCH}_core/profiled ${OS}_${ARCH}_core/product : $(buildATree)
-	sh $(buildATree) core ${GAMMADIR} ${OS} ${ARCH}
+	bash $(buildATree) core ${GAMMADIR} ${OS} ${ARCH}
 
 debugcore: ${OS}_${ARCH}_core/debug
 	(cd ${OS}_${ARCH}_core/debug; $(MAKE))
